### DISCUSSION TITLE
Small Suggestion

### DISCUSSION THREAD

**Discussion by Niassamond1** [AdriaGual/marvel-snap-bot#28] [Ideas]
Hello, first i'd like to thank you for this amazing software, it works wonders. i made some tweeks for me and it got me to vibranium. 
Id like to suggest a way for the bot to check the spots where its winning / losing, so that it could retreat from snaps. i also would love to see a card priority option, for instance when playing kazoo the idea is never playing cards cost 1 on first turns, leave them all for last turn, and on other turns playing other cards for instance angela, bishop , kazar and blue marvel, and dumping all cost 1 at last turn.
When i first tried the bot it would play angela in bad spots creating angels with zero power , bishop on last turn, and all cost 1 whenever, so it lost a lot of matches.

**Comment by paperc07** [comment]
Well would you share your tweaks cause I would love to get to Vibranium.

**Reply by Niassamond1** [reply]
Sorry but all my tweeks are just for my deck, and cards, probably not a good idea for using with kazoo deck.

**Reply by paperc07** [reply]
Come on! I have all the cards and I would like to try it. Help me out.

**Comment by Niassamond1** [comment]
Also, if possible it would be lovely to be able to set some cards priority to certain terrains, for instance cards that get stronger when destroyed or that cant be destroyed to be placed in terrains that destroy them. For instance in a death terrain or a bridge, the bot would primarely try to play armor, and if the bot checks that there's an armor on the terrain, it stops being afraid of throwing cards there.

**Comment by AdriaGual** [comment]
@Niassamond That sounds amazing! If you want, you can do a PR, I'll check it and merge it! (also, I really appreciate your feedback on how to play the game!)

Also the fields idea sounds great to me! (didn't know that I could play armor on a death terrain, you see, I spent more time creating the bot rather than playing the proper game haha)

**Reply by Niassamond1** [reply]
Ok, im a complete idiot when it comes to programming python, but in the weekend when i have some time i'll try something.

**Comment by ZloyPupsic** [comment]
@Niassamond can you share your tweaks and deck here for now, please?

**Reply by paperc07** [reply]
that's what im saying
